assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -55,25 +57,18 @@ import SavedIdeas from './SavedIdeas';
 import DraftIdeas from './DraftIdeas';
 import Empty from './Empty';
 import Footer from './Footer';
+import withMemoryRouter from '../../../../../components/withMemoryRouter';
+import Link from '../../../../../components/Link';
+import useQueryArg from '../../../../../hooks/useQueryArg';
 const { useSelect } = Data;
 
-function getHash( hash ) {
-	return hash ? hash.replace( '#', '' ) : false;
-}
-
-function isValidHash( hash ) {
-	return getHash( hash ) in DashboardIdeasWidget.tabToIndex;
-}
-
-function getIdeaHubContainerOffset( widgetOffset ) {
-	const header =
+const getIdeaHubContainerOffset = ( ideaHubWidgetOffsetTop ) => {
+	const siteHeaderHeight =
 		document.querySelector( '.googlesitekit-header' )?.offsetHeight || 0;
-	const adminBar = document.getElementById( 'wpadminbar' )?.offsetHeight || 0;
```

Comment:
The height of the admin bar is relevant here, but only at `window.innerWidth > 600` which is when it becomes fixed and our CSS adds a `top` rule for the SK header.

I'm guessing it doesn't matter though as long as the SK header is positioned properly relative to it?

Reply:
Fixed this as it wasn't accounted for 👍